cli/operator/create_slashing_protection_db.go

```diff
@@ -0,0 +1,151 @@
+package operator
```

Comment:
@moshe-blox Running a CLI command as a prerequisite sounds pretty harsh to me in terms of UX. maybe we can generate it on our on if it doesn't exist? of course we'll never want to overwrite existing db. so either newly create or point by cli param? IMO we need some more thought on this.

Reply:
So if we're going through with this, this should be represented in our README and docs. @olegshmuelov , @moshe-blox , @lior-blox , @arielssv
we should probably have a diagram or something about how do we want to act in any of the cases (db exists, db not exist and more) also from product perspective ( @arielssv )and make sure our code is aligned with it.